inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AbstractService.java

```diff
@@ -102,7 +102,9 @@ public  void run() {
                         }
                         count++;
                     }
-                    batchInsertEntities(entryList);
+                    if (entryList.size() > 0) {
```

Comment:
Its recommanded use `CollectionUtils.isNotEmpty` to check whether a collection is empty or not.

Reply:
CollectionUtils.isNotEmpty  need check more than once.
like this :
coll == null || coll.isEmpty();
 and here entryList is never be null. so check entryList.size()  will be beset.